zk-tutorials/proof-system/groth16/theory_to_practice.md

```diff
@@ -99,7 +99,7 @@ $A1 = \alpha_1 + w_0A_0(\tau)_1 + w_1A_1(\tau)_1 + w_2A_2(\tau)_1 + w_3A_3(\tau)
 
 $A_i(\tau)\cdot g_1$ can be calculated from the coefficients of $A_i(x)$ by multiplying each coefficient with the corresponding term $g_1, \tau\cdot g_1, \tau^2\cdot g_1, \ldots$, which are points made available by the CRS (common reference string).
 
-* $B2$ is a point in G2:
+* $B2$ is a point in G2: [skalman](# "your definition here is recursive you have B_2 on both side, you really need a better notation")
```

Comment:
Here the left notation is $B2$ and the right notation is $B_2$, so they are different.